tests/html5skin-test.js

```diff
@@ -126,6 +111,7 @@ describe('Controller', function() {
 
   describe('Closed Captions', function() {
     beforeEach(function() {
+      controller.createPluginElements();
```

Comment:
Not super important, but I think it might be simpler if we added this to the global `beforeEach` at the controller level instead of to each individual test group

Reply:
There are some tests that require ```createPluginElements``` to be called after some setup, like https://github.com/ooyala/html5-skin/pull/1111/files#diff-63a4cbb7eb24c1dc5f84f0a7439d0d34R1245 the 'Show player controls over ads' tests.

Reply:
Ok, got it